### PR TITLE
Enable more lint rules! (Fixes #195)

### LINKED ISSUE #195: Enable more lint rules
The lint rules on the bottom of [`.eslintrc`](https://github.com/facebook/react-devtools/blob/master/.eslintrc) should probably all be enabled. Not sure we want the `const` one.

### PR DESCRIPTION
**Pull request by flipstewart**
shocked

### CODE REVIEW

File: agent/Bridge.js [hunk 1]
```diff
@@ -323,16 +323,13 @@ class Bridge {
     if (!this._callers[name]) {
       return console.warn('unknown call');
     }
-    var args = args;
-    if (!Array.isArray(args)) {
-      args = [args];
-    }
+    var preparedArgs = !Array.isArray(args) ? [args] : args;
```
Comment (kassens) [inline agent/Bridge.js]: You could just overwrite `args` here when you remove the `var ` prefix.

File: .eslintrc [hunk 2]
```diff
@@ -46,9 +46,9 @@ rules:
   react/wrap-multilines: [2, {declaration: false, assignment: false}]
 
   # TODO: we might want these, but currently have multiple warnings
```
Comment (kassens) [inline .eslintrc]: You can remove this now :smile:

File: frontend/decorate.js [hunk 3]
```diff
@@ -130,14 +130,14 @@ module.exports = function (options: Options, Component: any): any {
 function arrayDiff(array, oldArray) {
   var names = new Set();
   var missing = [];
-  for (var i = 0; i < array.length; i++) {
-    names.add(array[i]);
+  for (var ai = 0; ai < array.length; ai++) {
```
Comment (kassens) [inline frontend/decorate.js]: If flow supported `let` already, we could use that here...

Otherwise, I'd prefer `i`, `j`, `k` as more traditional.

Comment (flipstewart) [inline frontend/decorate.js]: :+1:

File: .eslintrc (hunk 2 above)
Comment (flipstewart) [inline .eslintrc]: :joy:

File: agent/Bridge.js (hunk 1 above)
Comment (flipstewart) [inline agent/Bridge.js]: :+1:

**Comment by kassens** [comment]
This is great. Thanks!

**Comment by flipstewart** [comment]
@kassens happy to help!

**Pull request merged**